tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -238,15 +249,31 @@ def _opaque_to_cu_canonical(self, opaque_param):
       2 list for weights and biases respectively.
     """
     with ops.device("/gpu:0"):
-      weights, biases = gen_cudnn_rnn_ops.cudnn_rnn_params_to_canonical(
-          num_layers=self._num_layers,
-          num_units=self._num_units,
-          input_size=self._input_size,
-          params=opaque_param,
-          num_params=self._num_params,
-          rnn_mode=self._rnn_mode,
-          input_mode=self._input_mode,
-          direction=self._direction)
+      if self._num_proj:
+        num_params_weights = (self._num_params +
+                              1 * self._num_layers * self._num_dirs)
+        num_params_biases = self._num_params
+        weights, biases = gen_cudnn_rnn_ops.cudnn_rnn_params_to_canonical_v2(
```

Comment:
These should be guarded with https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/compat/compat.py for forward compatibility